### PR TITLE
Ports/jfduke3d: Add post_install() section + game icon + install into /usr/local/share/games/jfduke3d

### PR DESCRIPTION
**Pull request by kennethmyhra**
* Ports/jfduke3d: Install into '/usr/local/share/games/jfduke3d'
* Ports/jfduke3d: Add game icon
* Ports/jfduk3d: Add post_install() section
  * Adds a post_install() section printing out useful information on how to install the game files.

### CODE REVIEW

File: Ports/jfduke3d/package.sh [hunk 1]
```diff
@@ -6,6 +6,7 @@ install_dir='/usr/local/share/games/jfduke3d'
 launcher_name='Duke Nukem 3D'
 launcher_category='Games'
 launcher_command="${install_dir}/duke3d"
+icon_file="${SERENITY_INSTALL_ROOT}${install_dir}/game_icon.ico"
```
Comment (gmta) [inline Ports/jfduke3d/package.sh:9]: You don't need to copy the icon file into the filesystem; providing an `icon_file` will add the right sections to the binary pointed at by `launcher_command`.

File: Ports/jfduke3d/package.sh [hunk 2]
```diff
@@ -44,3 +44,14 @@ install() {
     run cp duke3d "${SERENITY_INSTALL_ROOT}${install_dir}"
     run cp rsrc/game_icon.ico "${SERENITY_INSTALL_ROOT}${install_dir}"
 }
+
+post_install() {
+  echo
+  echo 'Duke Nukem 3D is installed!'
+  echo
+  echo 'Make sure your game files are present in the installation directory:'
+  echo "    Inside SerenityOS: ~/.jfduke3d"
```
Comment (gmta) [inline Ports/jfduke3d/package.sh:53]: These can be single quotes

File: Ports/jfduke3d/package.sh [hunk 3]
```diff
@@ -44,3 +44,14 @@ install() {
     run cp duke3d "${SERENITY_INSTALL_ROOT}${install_dir}"
```
Comment (gmta) [inline Ports/jfduke3d/package.sh:44]: Typo in commit title: jfduk3d

**Comment by kennethmyhra** [comment]
Thanks Jelle! Should be fixed up

**Review by gmta — APPROVED** [review]

**Pull request merged**